## `nixpkgs-review` result

Generated using [`nixpkgs-review-gha`](https://github.com/Defelo/nixpkgs-review-gha)

Command: `nixpkgs-review pr 431019`

Logs: https://github.com/GaetanLepage/nixpkgs-review-gha/actions/runs/16733707174


---
### `x86_64-linux`
<details>
  <summary>:x: 38 packages failed to build:</summary>
  <ul>
    <li>python312Packages.fastmri</li>
    <li>python312Packages.fastmri.dist</li>
    <li>python312Packages.finetuning-scheduler</li>
    <li>python312Packages.finetuning-scheduler.dist</li>
    <li>python312Packages.gluonts</li>
    <li>python312Packages.gluonts.dist</li>
    <li>python312Packages.lightning</li>
    <li>python312Packages.lightning-utilities</li>
    <li>python312Packages.lightning-utilities.dist</li>
    <li>python312Packages.lightning.dist</li>
    <li>python312Packages.pyannote-audio</li>
    <li>python312Packages.pyannote-audio.dist</li>
    <li>python312Packages.pytorch-lightning</li>
    <li>python312Packages.pytorch-lightning.dist</li>
    <li>python312Packages.torchmetrics</li>
    <li>python312Packages.torchmetrics.dist</li>
    <li>python312Packages.whisperx</li>
    <li>python312Packages.whisperx.dist</li>
    <li>python313Packages.fastmri</li>
    <li>python313Packages.fastmri.dist</li>
    <li>python313Packages.finetuning-scheduler</li>
    <li>python313Packages.finetuning-scheduler.dist</li>
    <li>python313Packages.gluonts</li>
    <li>python313Packages.gluonts.dist</li>
    <li>python313Packages.lightning</li>
    <li>python313Packages.lightning-utilities</li>
    <li>python313Packages.lightning-utilities.dist</li>
    <li>python313Packages.lightning.dist</li>
    <li>python313Packages.pyannote-audio</li>
    <li>python313Packages.pyannote-audio.dist</li>
    <li>python313Packages.pytorch-lightning</li>
    <li>python313Packages.pytorch-lightning.dist</li>
    <li>python313Packages.torchmetrics</li>
    <li>python313Packages.torchmetrics.dist</li>
    <li>whisperx (python313Packages.whisperx)</li>
    <li>whisperx.dist (python313Packages.whisperx.dist)</li>
    <li>whisper-ctranslate2</li>
    <li>whisper-ctranslate2.dist</li>
  </ul>
</details>

---
<details>
<summary>Error logs: `x86_64-linux`</summary>
<details>
<summary>python312Packages.lightning-utilities</summary>
<pre>0.01s call     tests/unittests/core/test_imports.py::test_get_dependency_min_version_spec
0.01s setup    tests/unittests/cli/test_command_line.py::TestRequirements::test_requirements_set_oldest[positional]
0.01s setup    tests/scripts/test_adjust_torch_versions.py::test_adjust_torch_versions_call[optional]
0.01s setup    src/lightning_utilities/cli/dependencies.py::lightning_utilities.cli.dependencies._replace_package_name
0.01s teardown src/lightning_utilities/core/imports.py::lightning_utilities.core.imports.module_available
0.01s setup    src/lightning_utilities/core/enums.py::lightning_utilities.core.enums.StrEnum
0.01s teardown tests/unittests/core/test_imports.py::test_torch_class_raised
0.01s setup    tests/unittests/cli/test_command_line.py::TestRequirements::test_requirements_prune_pkgs[positional]
0.01s call     tests/unittests/cli/test_dependencies.py::test_prune_packages_txt
0.01s call     tests/unittests/core/test_imports.py::test_lazy_import
=========================== short test summary info ============================
FAILED tests/scripts/test_adjust_torch_versions.py::test_adjust_torch_versions_call[optional] - assert 1 == 0
FAILED tests/unittests/cli/test_command_line.py::test_version - assert 1 == 0
FAILED tests/unittests/cli/test_command_line.py::TestRequirements::test_requirements_prune_pkgs[positional] - assert 1 == 0
FAILED tests/unittests/cli/test_command_line.py::TestRequirements::test_requirements_prune_pkgs[optional] - assert 1 == 0
FAILED tests/unittests/cli/test_command_line.py::TestRequirements::test_requirements_set_oldest[positional] - assert 1 == 0
FAILED tests/unittests/cli/test_command_line.py::TestRequirements::test_requirements_set_oldest[optional] - assert 1 == 0
FAILED tests/unittests/cli/test_command_line.py::TestRequirements::test_requirements_replace_pkg[positional] - assert 1 == 0
FAILED tests/unittests/cli/test_command_line.py::TestRequirements::test_requirements_replace_pkg[optional] - assert 1 == 0
============ 8 failed, 51 passed, 4 deselected, 5 warnings in 4.17s ============</pre>
</details>
<details>
<summary>python313Packages.lightning-utilities</summary>
<pre>0.01s setup    tests/unittests/cli/test_command_line.py::TestRequirements::test_requirements_set_oldest[optional]
0.01s call     src/lightning_utilities/core/imports.py::lightning_utilities.core.imports.module_available
0.01s teardown tests/unittests/cli/test_dependencies.py::test_oldest_packages_pyproject_toml
0.01s call     tests/unittests/core/test_apply_func.py::test_application_to_collection_return_type[ori1-target1]
0.01s setup    tests/unittests/core/test_imports.py::testcompare_version
0.01s call     src/lightning_utilities/core/enums.py::lightning_utilities.core.enums.StrEnum
0.01s call     tests/unittests/core/test_imports.py::test_get_dependency_min_version_spec
0.01s setup    tests/unittests/core/test_apply_func.py::test_application_to_collection_return_type[ori2-target2]
0.01s call     tests/unittests/core/test_imports.py::test_requirement_cache_with_extras
0.01s setup    tests/scripts/test_adjust_torch_versions.py::test_adjust_torch_versions_call[mixed]
=========================== short test summary info ============================
FAILED tests/scripts/test_adjust_torch_versions.py::test_adjust_torch_versions_call[optional] - assert 1 == 0
FAILED tests/unittests/cli/test_command_line.py::test_version - assert 1 == 0
FAILED tests/unittests/cli/test_command_line.py::TestRequirements::test_requirements_prune_pkgs[positional] - assert 1 == 0
FAILED tests/unittests/cli/test_command_line.py::TestRequirements::test_requirements_prune_pkgs[optional] - assert 1 == 0
FAILED tests/unittests/cli/test_command_line.py::TestRequirements::test_requirements_set_oldest[positional] - assert 1 == 0
FAILED tests/unittests/cli/test_command_line.py::TestRequirements::test_requirements_set_oldest[optional] - assert 1 == 0
FAILED tests/unittests/cli/test_command_line.py::TestRequirements::test_requirements_replace_pkg[positional] - assert 1 == 0
FAILED tests/unittests/cli/test_command_line.py::TestRequirements::test_requirements_replace_pkg[optional] - assert 1 == 0
============ 8 failed, 51 passed, 4 deselected, 5 warnings in 4.15s ============</pre>
</details>
</details>


---
### `aarch64-linux`
<details>
  <summary>:x: 38 packages failed to build:</summary>
  <ul>
    <li>python312Packages.fastmri</li>
    <li>python312Packages.fastmri.dist</li>
    <li>python312Packages.finetuning-scheduler</li>
    <li>python312Packages.finetuning-scheduler.dist</li>
    <li>python312Packages.gluonts</li>
    <li>python312Packages.gluonts.dist</li>
    <li>python312Packages.lightning</li>
    <li>python312Packages.lightning-utilities</li>
    <li>python312Packages.lightning-utilities.dist</li>
    <li>python312Packages.lightning.dist</li>
    <li>python312Packages.pyannote-audio</li>
    <li>python312Packages.pyannote-audio.dist</li>
    <li>python312Packages.pytorch-lightning</li>
    <li>python312Packages.pytorch-lightning.dist</li>
    <li>python312Packages.torchmetrics</li>
    <li>python312Packages.torchmetrics.dist</li>
    <li>python312Packages.whisperx</li>
    <li>python312Packages.whisperx.dist</li>
    <li>python313Packages.fastmri</li>
    <li>python313Packages.fastmri.dist</li>
    <li>python313Packages.finetuning-scheduler</li>
    <li>python313Packages.finetuning-scheduler.dist</li>
    <li>python313Packages.gluonts</li>
    <li>python313Packages.gluonts.dist</li>
    <li>python313Packages.lightning</li>
    <li>python313Packages.lightning-utilities</li>
    <li>python313Packages.lightning-utilities.dist</li>
    <li>python313Packages.lightning.dist</li>
    <li>python313Packages.pyannote-audio</li>
    <li>python313Packages.pyannote-audio.dist</li>
    <li>python313Packages.pytorch-lightning</li>
    <li>python313Packages.pytorch-lightning.dist</li>
    <li>python313Packages.torchmetrics</li>
    <li>python313Packages.torchmetrics.dist</li>
    <li>whisperx (python313Packages.whisperx)</li>
    <li>whisperx.dist (python313Packages.whisperx.dist)</li>
    <li>whisper-ctranslate2</li>
    <li>whisper-ctranslate2.dist</li>
  </ul>
</details>

---
<details>
<summary>Error logs: `aarch64-linux`</summary>
<details>
<summary>python312Packages.lightning-utilities</summary>
<pre>0.01s call     tests/unittests/core/test_apply_func.py::test_apply_to_collection_allow_frozen_dataclass
0.01s setup    tests/unittests/core/test_inheritance.py::test_get_all_subclasses
0.01s setup    tests/scripts/test_adjust_torch_versions.py::test_adjust_torch_versions_call[mixed]
0.01s call     tests/unittests/core/test_imports.py::test_get_dependency_min_version_spec
0.01s call     src/lightning_utilities/core/imports.py::lightning_utilities.core.imports.requires
0.01s setup    tests/scripts/test_adjust_torch_versions.py::test_adjust_torch_versions_call[optional]
0.01s setup    tests/scripts/test_adjust_torch_versions.py::test_adjust_torch_versions_call[positional]
0.01s teardown tests/unittests/core/test_apply_func.py::test_apply_to_collection_allow_frozen_dataclass
0.01s teardown scripts/adjust-torch-versions.py::adjust-torch-versions._determine_torchaudio
0.01s teardown tests/unittests/cli/test_command_line.py::TestRequirements::test_requirements_replace_pkg[positional]
=========================== short test summary info ============================
FAILED tests/scripts/test_adjust_torch_versions.py::test_adjust_torch_versions_call[optional] - assert 1 == 0
FAILED tests/unittests/cli/test_command_line.py::test_version - assert 1 == 0
FAILED tests/unittests/cli/test_command_line.py::TestRequirements::test_requirements_prune_pkgs[positional] - assert 1 == 0
FAILED tests/unittests/cli/test_command_line.py::TestRequirements::test_requirements_prune_pkgs[optional] - assert 1 == 0
FAILED tests/unittests/cli/test_command_line.py::TestRequirements::test_requirements_set_oldest[positional] - assert 1 == 0
FAILED tests/unittests/cli/test_command_line.py::TestRequirements::test_requirements_set_oldest[optional] - assert 1 == 0
FAILED tests/unittests/cli/test_command_line.py::TestRequirements::test_requirements_replace_pkg[positional] - assert 1 == 0
FAILED tests/unittests/cli/test_command_line.py::TestRequirements::test_requirements_replace_pkg[optional] - assert 1 == 0
============ 8 failed, 51 passed, 4 deselected, 5 warnings in 3.06s ============</pre>
</details>
<details>
<summary>python313Packages.lightning-utilities</summary>
<pre>0.02s call     tests/unittests/cli/test_dependencies.py::test_oldest_packages_txt
0.01s setup    tests/unittests/cli/test_dependencies.py::test_prune_packages_txt
0.01s call     tests/unittests/core/test_enums.py::test_create_from_string
0.01s teardown tests/unittests/cli/test_dependencies.py::test_replace_packages_txt
0.01s call     tests/unittests/cli/test_dependencies.py::test_replace_packages_txt
0.01s setup    tests/unittests/cli/test_command_line.py::TestRequirements::test_requirements_replace_pkg[optional]
0.01s setup    src/lightning_utilities/core/imports.py::lightning_utilities.core.imports.module_available
0.01s setup    tests/unittests/cli/test_command_line.py::TestRequirements::test_requirements_set_oldest[positional]
0.01s call     scripts/adjust-torch-versions.py::adjust-torch-versions._determine_torchaudio
0.01s setup    tests/scripts/test_adjust_torch_versions.py::test_adjust_torch_versions_call[optional]
=========================== short test summary info ============================
FAILED tests/scripts/test_adjust_torch_versions.py::test_adjust_torch_versions_call[optional] - assert 1 == 0
FAILED tests/unittests/cli/test_command_line.py::test_version - assert 1 == 0
FAILED tests/unittests/cli/test_command_line.py::TestRequirements::test_requirements_prune_pkgs[positional] - assert 1 == 0
FAILED tests/unittests/cli/test_command_line.py::TestRequirements::test_requirements_prune_pkgs[optional] - assert 1 == 0
FAILED tests/unittests/cli/test_command_line.py::TestRequirements::test_requirements_set_oldest[positional] - assert 1 == 0
FAILED tests/unittests/cli/test_command_line.py::TestRequirements::test_requirements_set_oldest[optional] - assert 1 == 0
FAILED tests/unittests/cli/test_command_line.py::TestRequirements::test_requirements_replace_pkg[positional] - assert 1 == 0
FAILED tests/unittests/cli/test_command_line.py::TestRequirements::test_requirements_replace_pkg[optional] - assert 1 == 0
============ 8 failed, 51 passed, 4 deselected, 5 warnings in 2.76s ============</pre>
</details>
</details>


---
### `x86_64-darwin` (sandbox = true)
<details>
  <summary>:x: 38 packages failed to build:</summary>
  <ul>
    <li>python312Packages.fastmri</li>
    <li>python312Packages.fastmri.dist</li>
    <li>python312Packages.finetuning-scheduler</li>
    <li>python312Packages.finetuning-scheduler.dist</li>
    <li>python312Packages.gluonts</li>
    <li>python312Packages.gluonts.dist</li>
    <li>python312Packages.lightning</li>
    <li>python312Packages.lightning-utilities</li>
    <li>python312Packages.lightning-utilities.dist</li>
    <li>python312Packages.lightning.dist</li>
    <li>python312Packages.pyannote-audio</li>
    <li>python312Packages.pyannote-audio.dist</li>
    <li>python312Packages.pytorch-lightning</li>
    <li>python312Packages.pytorch-lightning.dist</li>
    <li>python312Packages.torchmetrics</li>
    <li>python312Packages.torchmetrics.dist</li>
    <li>python312Packages.whisperx</li>
    <li>python312Packages.whisperx.dist</li>
    <li>python313Packages.fastmri</li>
    <li>python313Packages.fastmri.dist</li>
    <li>python313Packages.finetuning-scheduler</li>
    <li>python313Packages.finetuning-scheduler.dist</li>
    <li>python313Packages.gluonts</li>
    <li>python313Packages.gluonts.dist</li>
    <li>python313Packages.lightning</li>
    <li>python313Packages.lightning-utilities</li>
    <li>python313Packages.lightning-utilities.dist</li>
    <li>python313Packages.lightning.dist</li>
    <li>python313Packages.pyannote-audio</li>
    <li>python313Packages.pyannote-audio.dist</li>
    <li>python313Packages.pytorch-lightning</li>
    <li>python313Packages.pytorch-lightning.dist</li>
    <li>python313Packages.torchmetrics</li>
    <li>python313Packages.torchmetrics.dist</li>
    <li>whisperx (python313Packages.whisperx)</li>
    <li>whisperx.dist (python313Packages.whisperx.dist)</li>
    <li>whisper-ctranslate2</li>
    <li>whisper-ctranslate2.dist</li>
  </ul>
</details>

---
<details>
<summary>Error logs: `x86_64-darwin`</summary>
<details>
<summary>python312Packages.lightning-utilities</summary>
<pre>0.04s call     tests/unittests/cli/test_dependencies.py::test_oldest_packages_pyproject_toml
0.01s call     tests/unittests/core/test_imports.py::test_requirement_cache
0.01s call     scripts/adjust-torch-versions.py::adjust-torch-versions.find_latest
0.01s setup    tests/scripts/test_adjust_torch_versions.py::test_adjust_torch_versions_call[optional]
0.01s call     src/lightning_utilities/core/imports.py::lightning_utilities.core.imports.requires
0.01s setup    scripts/adjust-torch-versions.py::adjust-torch-versions._determine_torchaudio
0.01s call     tests/unittests/core/test_imports.py::test_get_dependency_min_version_spec
0.01s teardown tests/scripts/test_adjust_torch_versions.py::test_adjust_torch_versions_call[mixed]

(6 durations < 0.005s hidden.  Use -vv to show these durations.)
=========================== short test summary info ============================
FAILED tests/scripts/test_adjust_torch_versions.py::test_adjust_torch_versions_call[optional] - assert 1 == 0
FAILED tests/unittests/cli/test_command_line.py::test_version - assert 1 == 0
FAILED tests/unittests/cli/test_command_line.py::TestRequirements::test_requirements_prune_pkgs[positional] - assert 1 == 0
FAILED tests/unittests/cli/test_command_line.py::TestRequirements::test_requirements_prune_pkgs[optional] - assert 1 == 0
FAILED tests/unittests/cli/test_command_line.py::TestRequirements::test_requirements_set_oldest[positional] - assert 1 == 0
FAILED tests/unittests/cli/test_command_line.py::TestRequirements::test_requirements_set_oldest[optional] - assert 1 == 0
FAILED tests/unittests/cli/test_command_line.py::TestRequirements::test_requirements_replace_pkg[positional] - assert 1 == 0
FAILED tests/unittests/cli/test_command_line.py::TestRequirements::test_requirements_replace_pkg[optional] - assert 1 == 0
============ 8 failed, 51 passed, 4 deselected, 5 warnings in 6.28s ============</pre>
</details>
<details>
<summary>python313Packages.lightning-utilities</summary>
<pre>0.16s call     tests/scripts/test_adjust_torch_versions.py::test_adjust_torch_versions_call[optional]
0.05s call     tests/unittests/cli/test_dependencies.py::test_oldest_packages_pyproject_toml
0.03s call     src/lightning_utilities/core/imports.py::lightning_utilities.core.imports.requires
0.03s call     scripts/adjust-torch-versions.py::adjust-torch-versions._determine_torchvision
0.02s call     src/lightning_utilities/core/enums.py::lightning_utilities.core.enums.StrEnum
0.01s call     tests/unittests/core/test_imports.py::test_requirement_cache
0.01s setup    tests/scripts/test_adjust_torch_versions.py::test_adjust_torch_versions_call[optional]
0.01s call     tests/unittests/core/test_imports.py::test_get_dependency_min_version_spec

(7 durations < 0.005s hidden.  Use -vv to show these durations.)
=========================== short test summary info ============================
FAILED tests/scripts/test_adjust_torch_versions.py::test_adjust_torch_versions_call[optional] - assert 1 == 0
FAILED tests/unittests/cli/test_command_line.py::test_version - assert 1 == 0
FAILED tests/unittests/cli/test_command_line.py::TestRequirements::test_requirements_prune_pkgs[positional] - assert 1 == 0
FAILED tests/unittests/cli/test_command_line.py::TestRequirements::test_requirements_prune_pkgs[optional] - assert 1 == 0
FAILED tests/unittests/cli/test_command_line.py::TestRequirements::test_requirements_set_oldest[positional] - assert 1 == 0
FAILED tests/unittests/cli/test_command_line.py::TestRequirements::test_requirements_set_oldest[optional] - assert 1 == 0
FAILED tests/unittests/cli/test_command_line.py::TestRequirements::test_requirements_replace_pkg[positional] - assert 1 == 0
FAILED tests/unittests/cli/test_command_line.py::TestRequirements::test_requirements_replace_pkg[optional] - assert 1 == 0
============ 8 failed, 51 passed, 4 deselected, 5 warnings in 6.76s ============</pre>
</details>
</details>


---
### `aarch64-darwin` (sandbox = true)
<details>
  <summary>:x: 38 packages failed to build:</summary>
  <ul>
    <li>python312Packages.fastmri</li>
    <li>python312Packages.fastmri.dist</li>
    <li>python312Packages.finetuning-scheduler</li>
    <li>python312Packages.finetuning-scheduler.dist</li>
    <li>python312Packages.gluonts</li>
    <li>python312Packages.gluonts.dist</li>
    <li>python312Packages.lightning</li>
    <li>python312Packages.lightning-utilities</li>
    <li>python312Packages.lightning-utilities.dist</li>
    <li>python312Packages.lightning.dist</li>
    <li>python312Packages.pyannote-audio</li>
    <li>python312Packages.pyannote-audio.dist</li>
    <li>python312Packages.pytorch-lightning</li>
    <li>python312Packages.pytorch-lightning.dist</li>
    <li>python312Packages.torchmetrics</li>
    <li>python312Packages.torchmetrics.dist</li>
    <li>python312Packages.whisperx</li>
    <li>python312Packages.whisperx.dist</li>
    <li>python313Packages.fastmri</li>
    <li>python313Packages.fastmri.dist</li>
    <li>python313Packages.finetuning-scheduler</li>
    <li>python313Packages.finetuning-scheduler.dist</li>
    <li>python313Packages.gluonts</li>
    <li>python313Packages.gluonts.dist</li>
    <li>python313Packages.lightning</li>
    <li>python313Packages.lightning-utilities</li>
    <li>python313Packages.lightning-utilities.dist</li>
    <li>python313Packages.lightning.dist</li>
    <li>python313Packages.pyannote-audio</li>
    <li>python313Packages.pyannote-audio.dist</li>
    <li>python313Packages.pytorch-lightning</li>
    <li>python313Packages.pytorch-lightning.dist</li>
    <li>python313Packages.torchmetrics</li>
    <li>python313Packages.torchmetrics.dist</li>
    <li>whisperx (python313Packages.whisperx)</li>
    <li>whisperx.dist (python313Packages.whisperx.dist)</li>
    <li>whisper-ctranslate2</li>
    <li>whisper-ctranslate2.dist</li>
  </ul>
</details>

---
<details>
<summary>Error logs: `aarch64-darwin`</summary>
<details>
<summary>python312Packages.lightning-utilities</summary>
<pre>0.10s call     tests/unittests/cli/test_command_line.py::test_version
0.05s call     tests/unittests/cli/test_dependencies.py::test_oldest_packages_pyproject_toml
0.05s call     tests/scripts/test_adjust_torch_versions.py::test_adjust_torch_versions_call[positional]
0.04s call     tests/scripts/test_adjust_torch_versions.py::test_adjust_torch_versions_call[optional]
0.03s call     tests/scripts/test_adjust_torch_versions.py::test_adjust_torch_versions_call[mixed]
0.03s call     tests/unittests/core/test_apply_func.py::test_apply_to_collection_allow_frozen_dataclass
0.02s call     tests/unittests/core/test_imports.py::test_requirement_cache
0.01s call     tests/unittests/cli/test_dependencies.py::test_replace_packages_txt

(11 durations < 0.005s hidden.  Use -vv to show these durations.)
=========================== short test summary info ============================
FAILED tests/scripts/test_adjust_torch_versions.py::test_adjust_torch_versions_call[optional] - assert 1 == 0
FAILED tests/unittests/cli/test_command_line.py::test_version - assert 1 == 0
FAILED tests/unittests/cli/test_command_line.py::TestRequirements::test_requirements_prune_pkgs[positional] - assert 1 == 0
FAILED tests/unittests/cli/test_command_line.py::TestRequirements::test_requirements_prune_pkgs[optional] - assert 1 == 0
FAILED tests/unittests/cli/test_command_line.py::TestRequirements::test_requirements_set_oldest[positional] - assert 1 == 0
FAILED tests/unittests/cli/test_command_line.py::TestRequirements::test_requirements_set_oldest[optional] - assert 1 == 0
FAILED tests/unittests/cli/test_command_line.py::TestRequirements::test_requirements_replace_pkg[positional] - assert 1 == 0
FAILED tests/unittests/cli/test_command_line.py::TestRequirements::test_requirements_replace_pkg[optional] - assert 1 == 0
============ 8 failed, 51 passed, 4 deselected, 5 warnings in 1.58s ============</pre>
</details>
<details>
<summary>python313Packages.lightning-utilities</summary>
<pre>0.12s call     tests/unittests/cli/test_command_line.py::TestRequirements::test_requirements_set_oldest[positional]
0.10s call     tests/unittests/cli/test_command_line.py::TestRequirements::test_requirements_set_oldest[optional]
0.10s call     tests/unittests/cli/test_command_line.py::TestRequirements::test_requirements_replace_pkg[positional]
0.05s call     tests/scripts/test_adjust_torch_versions.py::test_adjust_torch_versions_call[positional]
0.04s call     tests/scripts/test_adjust_torch_versions.py::test_adjust_torch_versions_call[optional]
0.03s call     tests/scripts/test_adjust_torch_versions.py::test_adjust_torch_versions_call[mixed]
0.01s call     tests/unittests/cli/test_dependencies.py::test_oldest_packages_pyproject_toml
0.01s call     tests/unittests/core/test_imports.py::test_requirement_cache

(13 durations < 0.005s hidden.  Use -vv to show these durations.)
=========================== short test summary info ============================
FAILED tests/scripts/test_adjust_torch_versions.py::test_adjust_torch_versions_call[optional] - assert 1 == 0
FAILED tests/unittests/cli/test_command_line.py::test_version - assert 1 == 0
FAILED tests/unittests/cli/test_command_line.py::TestRequirements::test_requirements_prune_pkgs[positional] - assert 1 == 0
FAILED tests/unittests/cli/test_command_line.py::TestRequirements::test_requirements_prune_pkgs[optional] - assert 1 == 0
FAILED tests/unittests/cli/test_command_line.py::TestRequirements::test_requirements_set_oldest[positional] - assert 1 == 0
FAILED tests/unittests/cli/test_command_line.py::TestRequirements::test_requirements_set_oldest[optional] - assert 1 == 0
FAILED tests/unittests/cli/test_command_line.py::TestRequirements::test_requirements_replace_pkg[positional] - assert 1 == 0
FAILED tests/unittests/cli/test_command_line.py::TestRequirements::test_requirements_replace_pkg[optional] - assert 1 == 0
============ 8 failed, 51 passed, 4 deselected, 5 warnings in 1.49s ============</pre>
</details>
</details>

--- a/pkgs/development/python-modules/lightning-utilities/default.nix
+++ b/pkgs/development/python-modules/lightning-utilities/default.nix
@@ -7,6 +7,7 @@
   setuptools,
 
   # dependencies
+  jsonargparse,
   looseversion,
   packaging,
   tomlkit,
@@ -19,14 +20,14 @@
 
 buildPythonPackage rec {
   pname = "lightning-utilities";
-  version = "0.15.0";
+  version = "0.15.1";
   pyproject = true;
 
   src = fetchFromGitHub {
     owner = "Lightning-AI";
     repo = "utilities";
     tag = "v${version}";
-    hash = "sha256-4WCsaprtAvrKFbPCa1bZJzT0Fo5F9sMYT2A+bolEmwk=";
+    hash = "sha256-GQVd81PHtAY9nr+YSVQli91AxbXSQFH0IALcZBMiu5o=";
   };
 
   postPatch = ''
@@ -37,6 +38,7 @@ buildPythonPackage rec {
   build-system = [ setuptools ];
 
   dependencies = [
+    jsonargparse
     looseversion
     packaging
     tomlkit

